src/doc/gui_manual/Operators/OperatorTypes/PICS/LimitCycle_operator.rst

```diff
@@ -0,0 +1,196 @@
+.. _Limit Cycle operator:
+
+Limit Cycle operator
+~~~~~~~~~~~~~~~~~~~~
+
+The Limit Cycle Operator allows one to search for closed material lines (limit
```

Comment:
Use of `material` here potentially confusing. Does this refer to materails in a mesh and used in Boundary/FilledBoundary plot

Reply:
Yes, I immediately thought of the same thing. I've left it here for now as a remnant from the old docs, but I will need to clear this up.